app/Http/Controllers/PayloadController.php

```diff
@@ -124,7 +127,10 @@ public function update(PayloadUpdateRequest $request, Webhook $webhook, Payload
             }
 
             DB::commit();
-            $request->session()->flash('messageSuccess', 'This payload successfully updated');
+            $request->session()->flash('messageSuccess', [
+                'status' => 'Update success',
+                'message' => 'This payload successfully updated'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 132</br>Each line in an array declaration must end in a comma